USST-RADSAT-SK/software-and-command
# Added "Test" build config

See Issues #87 

Added the `Test` build configuration, and set the necessary project settings to make it all work. Also added a basic Test Suite framework; not sure how much it'll be used or if it'll always look like that, but it's one way of doing it.

**To test and approve this PR:**
1. Close Eclipse
2. Checkout the branch `admin/test-build-config`
3. Start Eclipse
4. Build the Test configuration (hit the arrow next to the hammer icon, and select `Test` instead of `Debug`)
5. There should be no build errors
6. Go to `main.c` -> Lines wrapped in `#ifdef TEST` should NOT be greyed out (e.g. lines 21-23 and 86-92)
